Sources/MapboxNavigation/NavigationMapView+VanishingRouteLine.swift

```diff
@@ -339,4 +367,19 @@ extension NavigationMapView {
             return isMain ? trafficUnknownColor : alternativeTrafficUnknownColor
         }
     }
+    
+    func routeLineRestrictionStops(_ restrictionFeatures: [Turf.Feature], fractionTraveled: Double) -> [Double: UIColor] {
```

Comment:
Should it be named similar to the `routeLineCongestionGradient(_:fractionTraveled:isMain:isSoft:)`?

Reply:
Renamed